src/webgpu/api/validation/queue/copyToTexture/CopyExternalImageToTexture.spec.ts

```diff
@@ -20,7 +20,6 @@ import {
 import { kResourceStates } from '../../../../gpu_test.js';
 import {
   CanvasType,
-  canCopyFromCanvasContext,
```

Comment:
The `source_offscreenCanvas,contexts` also needs an update.

Reply:
Please verify this test works correctly after my change

Reply:
Checked these tests, and they can pass